### ISSUE TITLE
dhtnode and dhtdump binaries should be installed to /usr/bin not /usr/sbin

### ISSUE THREAD

**Issue by joseph-wakeling-sociomantic** [sociomantic-tsunami/dhtnode#151]
The current `dhtnode.pkg` spec installs the `dhtnode` and `dhtdump` binaries to `/usr/sbin`, which is reserved for system binaries; they should be installed into /usr/bin where ordinary command binaries live.

This will require tweaking both the `.pkg` file and the systemd service files.

**Comment by joseph-wakeling-sociomantic** [comment]
The same issue applies to dmqnode and dlsnode; I'll leave it to you to decide if you care about opening separate issues.

**Comment by joseph-wakeling-sociomantic** [comment]
> I heard a rumour that `usr/sbin` is for apps that require sudo permissions.

Yes, that matches my understanding.  Does dhtnode require those permissions?

**Comment by gavin-norman-sociomantic** [comment]
Yes. As far as I remember, [locking its memory into RAM](https://github.com/sociomantic-tsunami/dhtnode/blob/master/src/dhtnode/main.d#L524-L556) requires this.

**Comment by joseph-wakeling-sociomantic** [comment]
Ah, that makes sense.  Well, let's wait for @leandro-lucarella-sociomantic's pronouncement, but that sounds like a reasonable basis for the decision.
